### PR TITLE
Resize transformer fix

### PR DESCRIPTION
**Pull request by Fiddl3**
Many changes, please look at commit comments.
I tried to commit small code changes related to one issue at time.

ResizeTransformer should now work more smoothly, because layout is redrawn only once in one pass of Drag Listener.

I removed some unnecessary methods an simplified Transformers code...

### CODE REVIEW

**Comment by pedrovgs** [comment]
Nice PR, can you update this branch with pedrovgs:develop?

**Comment by Fiddl3** [comment]
Travis say `Could not find com.android.support:support-v4:21.0.2.` xD 
There is no support library on maven central (I think is in local repo, when you install android google support libs via SDK Manager)

**Comment by pedrovgs** [comment]
xD Revert your build.gradle changes to use an older version. I'll update that dependencies later.

File: draggablepanel/src/main/java/com/github/pedrovgs/DraggableViewCallback.java [hunk 1]
```diff
@@ -26,10 +26,10 @@
  */
 class DraggableViewCallback extends ViewDragHelper.Callback {
 
-  private static final int MINIMUN_DX_FOR_HORIZONTAL_DRAG = 5;
+  private static final int MINIMUM_DX_FOR_HORIZONTAL_DRAG = 10;
   private static final int MINIMUM_DY_FOR_VERTICAL_DRAG = 15;
-  private static final float X_MIN_VELOCITY = 1300;
-  private static final float Y_MIN_VELOCITY = 1300;
+  private static final float X_MIN_VELOCITY = 1500;
+  private static final float Y_MIN_VELOCITY = 1000;
```
Comment (pedrovgs) [inline draggablepanel/src/main/java/com/github/pedrovgs/DraggableViewCallback.java:32]: Why have you changed X_MIN_VELOCITY, Y_MIN_VELOCITY and MINIMUN_DX_FOR_HORIZONTAL_DRAG?

File: sample/build.gradle [hunk 2]
```diff
@@ -8,10 +8,10 @@ dependencies {
   compile 'com.squareup.dagger:dagger-compiler:1.1.0'
   compile 'com.jakewharton:butterknife:4.0.1'
   compile 'com.squareup.dagger:dagger:1.1.0'
-  compile 'com.google.android:youtube-android-player-api:1.0.0'
```
Comment (pedrovgs) [inline sample/build.gradle:11]: Can you update README.md to remove the section related to how to install this dependency into your local maven repository?

**Comment by Fiddl3** [comment]
`MINIMUN_DX_FOR_HORIZONTAL_DRAG` typo in `MINIMU``N` word 

`...MIN_VELOCITY` - I've tested various values and this to works ok for me, Y_MIN_VELOCITY was too high for landscape mode.

**Comment by pedrovgs** [comment]
Can you restore the value of MINIMUN_DX_FOR_HORIZONTAL_DRAG to 5?

**Comment by Fiddl3** [comment]
hmmmm.... I'm thinking `MINIMUM_DX` & `MINIMUM_DY` should be density dependent...

**Comment by pedrovgs** [comment]
Yes, should be density dependent. We can do that in other PR.

**Comment by pedrovgs** [comment]
There is something wrong with tv shows sample, ScaleTransformer and this branch :S. Here you have a video https://drive.google.com/file/d/0B9xkpTnF9BXjbDZQcU4tR1ZfblE/view?usp=sharing Can you take a look. This is the last fix needed to merge this PR :)

**Comment by Fiddl3** [comment]
Whats wrong there? I don't know where I should look

**Comment by Fiddl3** [comment]
aaa I see, second view content is missing, I'll take look at this

**Comment by pedrovgs** [comment]
Review also how you measure your max bottom position. I've changed your `<item name="y_scale_factor" format="float" type="dimen">2</item>` to `<item name="y_scale_factor" format="float" type="dimen">2.3</item>` and is working as you can see in this video: https://drive.google.com/file/d/0B9xkpTnF9BXjZndQOE9vSDNCa0E/view?usp=sharing

**Comment by Fiddl3** [comment]
travis doesn't like me & lastest google support library....
Travis downloads and installs support library, but then it doesn't see it... https://travis-ci.org/pedrovgs/DraggablePanel/builds/43138658#L935

**Comment by pedrovgs** [comment]
Thanks @Fiddl3 for this pull request!!!! I'm going to update project readme with a contributors section and you will be there :) What's your name and last name?

**Comment by Fiddl3** [comment]
No problem :) 
I'll continue working on lanscape support when I'll have time
I'm Rafał Pluta

**Comment by pedrovgs** [comment]
If you are using this library in any project, please send me an email with
the project name and link to update README.md file with that information.

2014-12-06 14:20 GMT+01:00 Fiddl3 notifications@github.com:

**Comment by Fiddl3** [comment]
This project is still in dev stage, I still have problems with rtmp stream playback...

**Comment by pedrovgs** [comment]
Ok, let me know once be published! Good luck ;)

**Comment by Fiddl3** [comment]
I will, thanks.

**Pull request merged**